Library/Formula/maven.rb

```diff
@@ -1,8 +1,8 @@
 class Maven < Formula
   homepage "https://maven.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  mirror "https://archive.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  sha256 "8c190264bdf591ff9f1268dc0ad940a2726f9e958e367716a09b8aaa7e74a755"
+  url "https://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz"
+  mirror "https://www.us.apache.org/dist/maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz"
```

Comment:
It seemed `archive.apache.org/dist` is not guaranteed to have the current release.
> This site contains the historical archive of old software releases.
> For current releases, please visit the mirrors.
> -- archive.apache.org/dist

And since `apache.org/dyn/closer.cgi` already provides the closest mirror–one which could also fail–it seemed logical to use a backup mirror guaranteed to have the current release instead.
> Please use the backup mirrors...if no other mirrors are working. 
> -- apache.org/dyn/closer.cgi (under Backup Sites)